tests/test_multistrategy_dispatch.py

```diff
@@ -0,0 +1,50 @@
+from cattr.multistrategy_dispatch import MultiStrategyDispatch
+
+
+class Foo(object):
+    pass
+
+
+def _fallback():
+    pass
+
+
+def _foo_func():
+    pass
+
+
+def _foo_cls():
+    pass
+
+
+def test_multistrategy_dispatch_register_cls():
+    _fallback()
```

Comment:
If I didn't add these calls, the coverage wasn't sufficient. 

I'm not sure how to tackle that, so I just call them.